securedrop/models.py

```diff
@@ -35,6 +37,14 @@
 ARGON2_PARAMS = dict(memory_cost=2**16, rounds=4, parallelism=2)
 
 
+def generate_otp_secret() -> str:
+    """
+    Generate an OTP secret of 160 bits encoded base32
+    """
+    symbols = string.ascii_uppercase + string.digits
```

Comment:
D'oh, my bad - this is 36 symbols. Currently the secret keys being generated use [ascii_upper]+[2.3,4,5,6,7], which makes sense in terms of readability (don't wanna mix up 0 and O or 1 and l).

Reply:
(This plus alembic migration woes covers off the app-test failures in CI.)

Reply:
Correct, this is what pyotp does ABCDEFGHIJKLMNOPQRSTUVWXYZ234567

(p.s. you are right that this is common idea that this would help usability, but think who speculated on this didnt really evaluate end users; in my opinion end users wont know anyway that we are not using 0 ad 1 and will confuse O with 0 and 1 with I anyway)

Reply:
@zenmonkeykstop: Could ABCDEFGHJKLMNPQRSTUVWXYZ23456789' work better as it does not include any of O 0 I 1 ?

Reply:
In relation to this change i've opened a pull on pyotp so to get to understand if any drawback could exist in this variation: https://github.com/pyauth/pyotp/pull/119

Reply:
Probably the first thing that comes to mind is interoperability with apps - just as an example google authenticator considers 8 and 9 illegal chars in secret keys.

Reply:
Nice catch. I was actually expecting this but did not find confirmations in the RFC about this limitation so i did not try.

Thank you @zenmonkeykstop .

I've corrected the patch accordingly.